### PR TITLE
[fix] fix errors in app.py

### PR DESCRIPTION
**Pull request by cdliang11**

### CODE REVIEW

**Comment by xingchensong** [comment]
plz fix lint

**Comment by cdliang11** [comment]
> plz fix lint

Error: ./runtime/web/app.py:4:1: F401 'torch' imported but unused.

For Windows and Linux, wenetruntime depends on torch. So import torch, but unused.

**Comment by xingchensong** [comment]
> > plz fix lint
> 
> Error: ./runtime/web/app.py:4:1: F401 'torch' imported but unused.
> 
> For Windows and Linux, wenetruntime depends on torch. So import torch, but unused.

we can simply set torch.seed

**Review by xingchensong — APPROVED** [review]

**Comment by cdliang11** [comment]
> > > plz fix lint
> > 
> > 
> > Error: ./runtime/web/app.py:4:1: F401 'torch' imported but unused.
> > For Windows and Linux, wenetruntime depends on torch. So import torch, but unused.
> 
> we can simply set torch.seed

Done.

**Comment by xingchensong** [comment]
good catch !
